bears/general/IndentationBear.py

```diff
@@ -386,6 +399,23 @@ def get_valid_sequences(file, sequence, annotation_dict):
                    lt_eq(sequence_position, comment.end)):
                     valid = False
 
+                if check_ending:
+                    if(comment.start.line == sequence_position.line and
+                            comment.end.line == sequence_position.line):
+                        sequence_line_text = sequence_line_text[
+                            :comment.start.column - 1] + sequence_line_text[
+                            comment.end.column-1:]
+
+            if encapsulators:
+                for encapsulator in encapsulators:
+                    if(gt_eq(sequence_position, encapsulator.start) and
```

Comment:
that should be in a new commit right? since it also deals with earlier lines of code?